File: .tmp_py4013/batch_04_sources/2469_convert_the_temperature.js
// LeetCode 2469 - Convert the Temperature
// https://leetcode.com/problems/convert-the-temperature/

/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function(celsius) {
    return [celsius + 273.15, celsius * 1.80 + 32.00];
};

File: .tmp_rs4013/rust_check_batch08/2469_convert_the_temperature.rs
struct Solution;
// LeetCode 2469 - Convert the Temperature
// https://leetcode.com/problems/convert-the-temperature/

impl Solution {
    pub fn convert_temperature(celsius: f64) -> Vec<f64> {
        vec![celsius + 273.15, celsius * 1.80 + 32.00]
    }
}

fn main() {}
